packages/manager/modules/support/src/ticket/ticket.scss

```diff
@@ -0,0 +1,7 @@
+.message {
+  background-color: transparent;
+  border: none;
+  margin: 0;
+  padding: 0;
+  word-break: normal;
```

Comment:
I think all those can be handled by bootstrap utilities

Reply:
We plan to address this in the second wave. Thanks for the inputs!